clang/test/CodeGenCXX/debug-info-structured-binding.cpp

```diff
@@ -2,10 +2,8 @@
 
```

Comment:
Please can you add `--implicit-check-not="call void @llvm.dbg.declare"` to the `FileCheck` command?

Reply:
Added the `--implicit-check-not` option.